getting-started/what-you-need.md

```diff
@@ -1,102 +1,118 @@
+To add these client assets, use either the [static assets](#using-static-assets) or the [CDN](#using-cdn) method.
 
-To have the Telerik Blazor components look and behave as expected in the browser, you need some assets. 
-@[template](/_contentTemplates/common/js-interop-file.md#app-paths)
+### Using Static Assets
 
+You can add the [Telerik JS Interop file](#telerik-js-interop-file) and the [Telerik Stylesheet](#telerik-stylesheet) as [static assets](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-6.0&tabs=visual-studio#consume-content-from-a-referenced-rcl). Static assets (the `_content` folder) are automatically included in the solution from the NuGet package during build, so all you need is to enable static assets as shown in the snippet below. The `_content` folder is expanded by the framework into the local NuGet cache, and the project copies it from there.
 
-#### The Telerik stylesheet - For more information, refer to the [Themes]({%slug general-information/themes%}) article. Here is a short example:
+To enable the use of static assets in your project, add the `app.UseStaticFiles();` line to the startup file of your **Server** project (by default, this line is already present):
 
-@[template](/_contentTemplates/common/js-interop-file.md#theme-static-asset-snippet)
+ * `Startup.cs` for .NET 3.x and .NET 5
+ * `Program.cs` for .NET 6
 
+**C#**
+@[template](/_contentTemplates/common/js-interop-file.md#enable-static-assets-snippet)
 
-#### The Telerik JS Interop file - Provides features that cannot be implemented with native Blazor.
+#### Telerik Stylesheet
 
-@[template](/_contentTemplates/common/js-interop-file.md#js-interop-file-snippet)
+The stylesheet allows you to use one of the built-in [Themes]({%slug general-information/themes%}), for example, the Default theme:
 
+@[template](/_contentTemplates/common/js-interop-file.md#theme-static-asset-snippet)
 
-### Static Assets
 
-You can add the Telerik JS Interop file as a [static asset](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.1&tabs=visual-studio#consume-content-from-a-referenced-rcl). Static assets (the `_content` folder) are automatically included in the solution from the NuGet package during build, so all you need is to reference the asset as shown in the snippet below. The `_content` folder is expanded by the framework into the local NuGet cache, and the project copies it from there.
+#### Telerik JS Interop File
 
-To enable the use of static assets in your project, add the following line to the startup file of your **Server** project:
+The JS Interop file provides features that cannot be implemented with native Blazor.
 
- * `Startup.cs` for .NET 3.x and .NET 5
- * `Program.cs` for .NET 6
+@[template](/_contentTemplates/common/js-interop-file.md#js-interop-file-snippet)
 
-**C#**
-@[template](/_contentTemplates/common/js-interop-file.md#enable-static-assets-snippet)
 
-### CDN
+### Using CDN
```

Comment:
A few years ago the guidance was to avoid -ing forms, has this changed?

Reply:
In titles and headings, we recommend using -ing forms. We avoid them in the free text because they can be ambiguous.

Reply:
indeed, that is new, from about a year ago. Thanks